test/controllers/AuthenticatedActionRefinerTest.scala

```diff
@@ -57,7 +58,7 @@ class AuthenticatedActionRefinerTest extends ControllerBaseSpec {
       "there is no active session" in new FakeController(Future.failed(NoActiveSessionException)) {
         val result: Future[Result] = onPageLoad()(request)
         status(result) shouldBe SEE_OTHER
-        redirectLocation(result) shouldBe Some(ApplicationConfig.ivLoginUrl)
+        redirectLocation(result).get shouldBe "/gg/sign-in"
```

Comment:
Can this be wrapped in a Some

Reply:
it could.. my personal preference for tests is .get I dont like Some's it makes me think the code is returning an option, where as redirectLocation is the option.. 
That way I can see exactly what the code should be returning

Reply:
Yes but wont the test fail for the wrong reason. None was not equal to Some("/gg/sign-in") as opposed to None.get exception

Reply:
Sorry other way around ^

Reply:
Have amended to follow suit, perhaps we should look to find a standard were happy with to push across future tests

Reply:
we could write our own redirectLocation() that returns a string or throws a failure stating there is no redirect locataion..

Reply:
Yup, that is what I meant :)